packages/office-ui-fabric-react/src/components/Button/docs/ButtonA11y.md

```diff
@@ -0,0 +1,12 @@
+Button is accessible if these considerations are followed:
+
+- Use a touch target area of `44px` by `44px`
```

Comment:
hmm. 44x44 is far larger than pretty much any button currently. A default button is 32px tall. This guidance is super confusing.